--- a/makefiles/Makefile.linux
+++ b/makefiles/Makefile.linux
@@ -22,7 +22,7 @@ AR=ar
 # To use an alternative text field widget for search/replace, add
 # -DXNE_TEXTFIELD to the CFLAGS.
 # 
-CFLAGS=-O -std=gnu99 -I/usr/X11R6/include -I/usr/include/X11 -DUSE_DIRENT -DUSE_LPR_PRINT_CMD $(shell pkg-config --cflags xft fontconfig)
+CFLAGS+=-O -std=gnu99 -I/usr/X11R6/include -I/usr/include/X11 -DUSE_DIRENT -DUSE_LPR_PRINT_CMD $(shell pkg-config --cflags xft fontconfig)
 
 ARFLAGS=-urs
 
